apisix/cli/ngx_tpl.lua

```diff
@@ -124,7 +124,7 @@ http {
 
 {% end %}
 
-{% if stream_proxy then %}
+{% if proxy_mode == "stream" or proxy_mode == "http&stream" then %}
```

Comment:
Could we use `enable_stream` here?

Reply:
We dont have `enable_stream` field in the configuration.

Reply:
We could pass the `enable_stream` like `enable_http`

Reply:
@Revolyssup

Reply:
@monkeyDluffy6017 Done.